index.html

```diff
@@ -173,126 +173,364 @@ <h2>
+            regardless of lock state.
+          </p>
+        </p>
+      </section>
+      <section>
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl data-dfn-for="PointerLockOptions" data-link-for=
+        "PointerLockOptions">
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `"pointerlockchange"` and `"pointerlockerror"` Events
+        </h3>
+        <p class="note">
+          Two events are used to communicate pointer lock state change or an
+          error in changing state. They are named <dfn>pointerlockchange</dfn>
+          and <dfn>pointerlockerror</dfn>.
+        </p>
+        <p class="note">
+          Magnification software increases the size of content on the screen.
+          It uses the mouse to move the magnified point of focus around. When a
+          pointer lock is initiated, the magnification software needs to switch
+          to using the keyboard to move the magnified point of focus around
+          instead. When a pointerlockchange event is fired, web browsers
+          therefore need to make sure the event is communicated to assistive
+          technologies like screen magnifiers.
+        </p>
+      </section>
+      <section>
+        <h3><dfn>Exit Pointer Lock</dfn></h3>
+        <p>
+          The process of exiting pointer lock is as follows:
+          <ol>
+            <li>
+              <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockchange</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a> on the
+                      <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                        User Interaction task source</a>
+            </li>
+            <li>
+              The system mouse cursor must be displayed again and positioned at
+              the same location that it was when pointer lock was entered (the
+              same location that is reported in <code>screenX</code>,
+              <code>screenY</code>, when the pointer is locked).
```

Comment:
This is duplicated in the definition of exitPointerLock().

Additionally, this would be best stored as an additional piece of state on the user agent, like pointer-lock target and pointer-lock options.

Reply:
Added a |cursor position| variable to the user agent and referenced that when entering and exiting pointer lock. I also removed this  from the exitPointerLock() method definition.